README.md

```diff
@@ -102,7 +102,7 @@ const unsignedClaim = {
   sub: "did:ethr:0x413daa771a2fc9c5ae5a66abd144881ef2498c54"
 }
 
-uport .createVerificationRequest(unsignedClaim)
+uport .requestSignVerification(unsignedClaim)
```

Comment:
Naming this `requestSignVerification` feels weird to me because it's `request[Verb: action for requested party to do][Noun: thing being requested from them]` instead of just `request[Noun: thing being requested from them]`.  I would prefer either `requestVerification` and justify by saying that being signed is inherent to the definition of what we're calling a "verification," or `requestVerificationSignature` to capture the fact that the claim is already provided and the client just needs to sign.

P.S. there is an extra space between `uport` and the function name.

Reply:
I like requestVerificationSignature, lets use that. The alternative requestVerification is very confusing for me, it sounds like the function to use if I want to ask a user to share a verification, which would be a SD request not this one

Reply:
I agree too, but it should also be consistent with uport-credentials.  If we want to go with `requestVerificationSignature` we should also have the credentials version be `createVerificationSignatureRequest` which is fairly verbose, but I'm still into it.